dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/h2/Http2TransportListener.java

```diff
@@ -16,4 +16,7 @@
  */
 package org.apache.dubbo.remoting.http12.h2;
 
-public interface Http2TransportListener extends CancelableTransportListener<Http2Header, Http2InputMessage> {}
+public interface Http2TransportListener extends CancelableTransportListener<Http2Header, Http2InputMessage> {
+
+    void onStreamClosed();
+}
```

Comment:
**Suggestion**: Document the expected behavior and usage of the onStreamClosed method within the interface to guide implementers. <br> **Code Suggestion**: 
 ```
 +/**
 + * Invoked when a stream is closed. Implementations should perform necessary cleanup.
 + */
 void onStreamClosed();
 ```